### ISSUE TITLE
Not working when site deployed to IIS

### ISSUE THREAD

**Issue by pvagnini** [emn178/Chart.PieceLabel.js#63]
Hi,

I have used this code on an Angular 6 ASP Core project. In my development environment (Angular CLI and VS2017) all works very well.

However when I deploy the code to an IIS server the chart labels are not displayed.

Do you have any suggestions please? Any help would be most appreciated.

Regards
Paolo

**Comment by steinv** [comment]
It is because the imports get messed up when you do a production build. Try like this:
import 'chart.js';
import 'chart.piecelabel.js';

**Comment by emn178** [comment]
do you solve it?

**Comment by pvagnini** [comment]
Yes I did. I did 2 things and I am not sure which one or both did the fix until I do a new production deploy.
I am using ng2-charts-x in my Angular project. 
1) added import 'chart.piecelabel.js'; along with the existing import 'chart.js'; to the index.d.ts file for ng2-charts-x the node_modules of my project. 
2) There is an import of {ChartsModule} relating to ng2-charts-x in the module.ts that relates to my project and I initially just placed the import 'chart.piecelabel.js'; there which worked in development but not when deployed. I then moved this import to the component.ts that directly relates to the page the charts are on.
